src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1 @@
+com.google.common.collect.Lists$TransformingRandomAccessList
```

Comment:
No, this is crazy. Just

```diff
-? Collections.unmodifiableList(resultsToCheck)
+? new ArrayList<>(resultsToCheck)
```

and the problem is solved.

Reply:
I don't remember why I have used `unmodifiableList`.
We can simply stop using it as jglick pointed.
Would you remove them from following constructors?
https://github.com/jenkinsci/matrix-combinations-plugin/blob/101d1cc9a83ea46c6a1e3028b88b5d6bbb45d551/src/main/java/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue.java
https://github.com/jenkinsci/matrix-combinations-plugin/blob/101d1cc9a83ea46c6a1e3028b88b5d6bbb45d551/src/main/java/hudson/plugins/matrix_configuration_parameter/shortcut/ResultShortcut.java